### PR TITLE
Adding a code sample of how the accordion widget can be used

### PR DESCRIPTION
**Pull request by eduard13**
## Purpose of this PR

This PR is an improvement of Accordion widget, which provides an example of how the accordion can be used and how the widget's options can be passed during initialization. 
Also the example's result preview is provided.

## Affected URLs

- https://devdocs.magento.com/guides/v2.1/javascript-dev-guide/widgets/widget_accordion.html
- https://devdocs.magento.com/guides/v2.2/javascript-dev-guide/widgets/widget_accordion.html
- https://devdocs.magento.com/guides/v2.3/javascript-dev-guide/widgets/widget_accordion.html

whatsnew
Added a new code sample showing how to use the [accordion widget](https://devdocs.magento.com/guides/v2.3/javascript-dev-guide/widgets/widget_accordion.html).

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.1/javascript-dev-guide/widgets/widget_accordion.md [hunk 1]
```diff
@@ -179,6 +179,47 @@ $( "#element" ).accordion( "deactivate", 1 );
...
+            <h4>Title 3</h4>
+        </div>
+    </div>
+    <div data-role="content">Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical Latin literature from 45 BC, making it over 2000 years old.</div>
+</div>
+```
+
+### Result
+
+The result is a three sections with separate swapped content.
```
Comment (jeff-matthews) [inline guides/v2.1/javascript-dev-guide/widgets/widget_accordion.md]:
```suggestion
The result is three sections with separate swapped content.
```

**Comment by eduard13** [comment]
Thank you @jeff-matthews, it was adjusted.

**Comment by jeff-matthews** [comment]
running tests

**Pull request merged**